src/jdk.compiler/share/man/serialver.md

```diff
@@ -60,15 +60,8 @@ arguments, the `serialver` command prints a usage line.
     application launcher. For example, `-J-Xms48m` sets the startup memory to
     48 MB.
 
-## Notes
+## Warning
```

Comment:
Should we move the warning more upfront?

Reply:
I thought it was sufficient to change the "Notes" heading to "Warning" and to leave it in place. The tool docs are styled after Unix "man pages" which typically have Name, Synopsis, Description, Options (sometimes the last two are swapped), and then possibly other sections if the doc is long, followed by things like Notes and Warnings at the end. Of course we needn't adhere to this structure slavishly. If it were sufficiently important we could diverge from the usual structure, but in this case it doesn't seem to me that the severity of the issue is so high that we need to place the warning more prominently.